src/main/java/com/vectorcast/plugins/vectorcastexecution/job/NewPipelineJob.java

```diff
@@ -329,6 +334,38 @@ private String getBaseJenkinsfile() throws IOException {
 	 * @return script portion of pipeline job.
 	 * @throws IOException
 	 */
+    public String getVersion() {
+        
+        String path = null;
```

Comment:
Can we initialize this variable inside the `try` block, like this:
`String path = getClass().getProtectionDomain().getCodeSource().getLocation().getPath();`
That way we reduce the variable's scope and can avoid setting it to null.

Reply:
If you can find a better/safer way to implement, that's awesome.  Please note that there is a similar block inside VectorCASTSetup.java.

Reply:
I was trying keep everything as localized as possible as I don't know what side effects there would be in trying to have VectorCASTSetup.java access something in NewPipelineJob.java and vice versa.  I also wasn't sure if there was a place both could access (BaseJob.java?) that would have access to the right data at the right time.